app/Randomizer.php

```diff
@@ -799,6 +801,7 @@ public function setTexts(Rom $rom) {
 			"A weeknight is\na tiny\nnobleman",
 			"The chimney\nsweep wore a\nsoot and tye.",
 			"Gardeners like\nto spring into\naction.",
+			"bad at nuclear\nphysics. I\nGot no fission",
```

Comment:
...and yet this bad is not capitalized?

Reply:
That's just how bad he is at nuclear physics.

Reply:
"Got no fission" is too long, but you can move got to the previous line I believe